src/machi_cr_client.erl

```diff
@@ -360,6 +360,9 @@ do_append_head3(Prefix, Chunk, ChunkExtra, Depth, STime, TO,
             %% written block is.  But we lost a race.  Repeat, with a new
             %% sequencer assignment.
             do_append_head(Prefix, Chunk, ChunkExtra, Depth, STime, TO, S);
+        {error, trimmed} = Err ->
+            %% TODO: behaviour
+            {reply, Err, S};
```

Comment:
When appending @ head, returning this error now is good.  I don't think it's a good idea to continue attempting to trim the middle(s) and tail, though I suppose that someone could argue that it would be worthwhile.